recipes/qt/6.x.x/conanfile.py

```diff
@@ -1196,7 +1196,10 @@ def _create_plugin(pluginname, libname, plugintype, requires):
                     self.cpp_info.components["QCocoaIntegrationPlugin"].frameworks = [
                         "AppKit", "Carbon", "CoreServices", "CoreVideo", "IOKit", "IOSurface", "Metal", "QuartzCore"
                     ]
-                elif self.settings.os in ["iOS", "tvOS"]:
+                if self.settings.os in ["Macos", "iOS"]:
+                    # https://github.com/qt/qtbase/blob/v6.5.3/src/gui/CMakeLists.txt#L963
```

Comment:
The linked source is for v6.5.3. Does this still apply in 6.7 and 6.8?

Reply:
Yes. https://github.com/qt/qtbase/blob/v6.7.3/src/gui/CMakeLists.txt#L463